label_studio/tests/data_manager/test_api_actions.py

```diff
@@ -137,24 +137,31 @@ def test_action_remove_duplicates(business_client, project_id, storage_model, li
     # task 4: add duplicated task, with storage link and one annotation
     task4 = make_task(task_data, project)
     make_annotation({'result': []}, task4.id)
+    # this task would have row_index=0 instead of None if it was created after multitask support was added
     link_model.objects.create(task=task4, key='duplicated.jpg', storage=storage)
 
+    # task 5: add not a duplicated task using the same key, ensuring multiple tasks in the same key don't interfere
```

Comment:
```suggestion
    # task 5: add a duplicated task using the same key, ensuring multiple tasks in the same key don't interfere
```

I assume this is a typo?

Reply:
yes :)